.gitpod.yml

```diff
@@ -1,28 +1,54 @@
 image:
   file: .gitpod.dockerfile
 tasks:
-  - init: npm_config_yes=true pnpm setup:project
+  - init: npm_config_yes=true pnpm setup:project && gp sync-done setup
     command: mkdir -p /workspace/data && mongod --dbpath /workspace/data 
-  - command: redis-server
+  - init: gp sync-await setup
+    command: redis-server
   - name: Shared Library
+    init: gp sync-await setup
     command: cd libs/shared && pnpm start
   - name: DAL
+    init: gp sync-await setup
     command: cd libs/dal && pnpm start
   - name: TESTING LIB
+    init: gp sync-await setup
     command: cd libs/testing && pnpm start
   - name: API
+    init: gp sync-await setup
     command: cd apps/api && pnpm start
   - name: WS
+    init: gp sync-await setup
     command: cd apps/ws && pnpm start
   - name: WEB
-    command: cd apps/web && pnpm start
+    init: gp sync-await setup
+    command: |
+      cd apps/web
+      API_URL=`gp url 3000`
```

Comment:
I think the WS_URL is missing on the web since the client tries to access localhost when visiting the web platform.